### ISSUE TITLE
Gemspec Doesn't declare pry dependency

### ISSUE THREAD

**Issue by schneems** [maerch/ruby-drawille#2]
The example doesn't work out of the box when pry isn't on on the load path: 

```
$ bundle exec irb
irb(main):001:0> require 'drawille'
LoadError: cannot load such file -- pry
    from /Users/richardschneeman/.gem/ruby/2.2.2/gems/drawille-0.3.2/lib/drawille/frameable.rb:3:in `require'
    from /Users/richardschneeman/.gem/ruby/2.2.2/gems/drawille-0.3.2/lib/drawille/frameable.rb:3:in `<top (required)>'
    from /Users/richardschneeman/.gem/ruby/2.2.2/gems/drawille-0.3.2/lib/drawille.rb:4:in `require'
    from /Users/richardschneeman/.gem/ruby/2.2.2/gems/drawille-0.3.2/lib/drawille.rb:4:in `<top (required)>'
    from (irb):1:in `require'
    from (irb):1
    from /Users/richardschneeman/.rubies/ruby-2.2.2/bin/irb:11:in `<main>'
irb(main):002:0>
irb(main):003:0* canvas = Drawille::Canvas.new
NameError: uninitialized constant Drawille::Canvas
    from (irb):3
    from /Users/richardschneeman/.rubies/ruby-2.2.2/bin/irb:11:in `<main>'
irb(main):004:0>
irb(main):005:0* (0..1800).step(10).each do |x|
irb(main):006:1*   canvas.set(x/10, 10 + Math.sin(x * Math::PI / 180) * 10)
irb(main):007:1> end
NoMethodError: undefined method `set' for nil:NilClass
    from (irb):6:in `block in irb_binding'
    from (irb):5:in `step'
    from (irb):5:in `each'
    from (irb):5
    from /Users/richardschneeman/.rubies/ruby-2.2.2/bin/irb:11:in `<main>'
irb(main):008:0>
irb(main):009:0* puts canvas.frame
NoMethodError: undefined method `frame' for nil:NilClass
    from (irb):9
    from /Users/richardschneeman/.rubies/ruby-2.2.2/bin/irb:11:in `<main>'
```

**Comment by maerch** [comment]
Ups!

I can reproduce this bug, but actually I have no idea why this is happening. I will investigate.

Thanks for opening this issue!

**Comment by maerch** [comment]
Ahhh, for some reason I haven't publish the most recent version. Will fix it! Thanks again!

**Comment by maerch** [comment]
Fixed in `0.3.3`
